iceoryx2-ffi/cxx/include/iox2/client.hpp

```diff
@@ -42,32 +46,63 @@ class Client {
     /// if the [`Server`](crate::port::server::Server)s buffer is full.
     auto unable_to_deliver_strategy() const -> UnableToDeliverStrategy;
 
-    /// Acquires an [`RequestMutUninit`] to store payload. This API shall be used
-    /// by default to avoid unnecessary copies.
-    auto loan_uninit()
-        -> iox::expected<RequestMutUninit<Service, RequestPayload, RequestHeader, ResponsePayload, ResponseHeader>,
-                         LoanError>;
+    /// Returns the maximum number of elements that can be loaned in a slice.
+    template <typename T = RequestPayload, typename = std::enable_if_t<iox::IsSlice<T>::VALUE, void>>
+    auto initial_max_slice_len() const -> uint64_t;
+
+    /// Copies the input value into a [`RequestMut`] and sends it. On success it
+    /// returns a [`PendingResponse`] that can be used to receive a stream of
+    /// [`Response`](crate::response::Response)s from the
```

Comment:
Sorry, I realized it just now. But could you please remove the pending `(...)` from the link since it follows a rust namespace and is invalid in C++. So ``[`Response`](crate::response::Response)`` should be ``[`Response`]``

Reply:
This goes for all the documentation in all headers in this PR. I am sorry :(